feature/experimental/bgp/otg_tests/bgp_remove_private_as/bgp_remove_private_as_otg_test.go

```diff
@@ -0,0 +1,417 @@
+// usable addresses. This does not apply to IPv6 which allows /127
+// for point to point links, but we use /126 so the numbering is
+// consistent with IPv4.
+
+const (
+	advertisedRoutesv4Net    = "203.0.113.1"
+	advertisedRoutesv4Prefix = 32
+	peerGrpName1             = "BGP-PEER-GROUP1"
+	peerGrpName2             = "BGP-PEER-GROUP2"
+	routeCount               = 254
+	dutAS                    = 500
+	ateAS1                   = 100
+	ateAS2                   = 200
+	plenIPv4                 = 30
+	plenIPv6                 = 126
+	removeASPath             = true
+)
+
+var (
+	dutSrc = attrs.Attributes{
+		Desc:    "DUT to ATE source",
+		IPv4:    "192.0.2.1",
+		IPv6:    "2001:db8::192:0:2:1",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	ateSrc = attrs.Attributes{
+		Name:    "ateSrc",
+		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::192:0:2:2",
+		MAC:     "02:00:01:01:01:01",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	dutDst = attrs.Attributes{
+		Desc:    "DUT to ATE destination",
+		IPv4:    "192.0.2.5",
+		IPv6:    "2001:db8::192:0:2:5",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	ateDst = attrs.Attributes{
+		Name:    "ateDst",
+		IPv4:    "192.0.2.6",
+		IPv6:    "2001:db8::192:0:2:6",
+		MAC:     "02:00:02:01:01:01",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+)
+
+// configureDUT configures all the interfaces on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	dc := gnmi.OC()
+	i1 := dutSrc.NewOCInterface(dut.Port(t, "port1").Name(), dut)
+	gnmi.Replace(t, dut, dc.Interface(i1.GetName()).Config(), i1)
+
+	i2 := dutDst.NewOCInterface(dut.Port(t, "port2").Name(), dut)
+	gnmi.Replace(t, dut, dc.Interface(i2.GetName()).Config(), i2)
+}
+
+// verifyPortsUp asserts that each port on the device is operating.
+func verifyPortsUp(t *testing.T, dev *ondatra.Device) {
+	t.Helper()
+	for _, p := range dev.Ports() {
+		status := gnmi.Get(t, dev, gnmi.OC().Interface(p.Name()).OperStatus().State())
+		if want := oc.Interface_OperStatus_UP; status != want {
+			t.Errorf("%s Status: got %v, want %v", p, status, want)
+		}
+	}
+}
+
+// bgpCreateNbr creates a BGP object with neighbors pointing to ateSrc and ateDst.
+func bgpCreateNbr(localAs, peerAs uint32, dut *ondatra.DUTDevice) *oc.NetworkInstance_Protocol {
+	nbr1v4 := &bgpNeighbor{as: ateAS1, neighborip: ateSrc.IPv4, isV4: true, peerGrp: peerGrpName1}
+	nbr2v4 := &bgpNeighbor{as: ateAS2, neighborip: ateDst.IPv4, isV4: true, peerGrp: peerGrpName2}
+	nbrs := []*bgpNeighbor{nbr1v4, nbr2v4}
+
+	d := &oc.Root{}
+	ni1 := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	ni_proto := ni1.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := ni_proto.GetOrCreateBgp()
+	global := bgp.GetOrCreateGlobal()
+	global.RouterId = ygot.String(dutDst.IPv4)
+	global.As = ygot.Uint32(localAs)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
```

Comment:
ATE version also does the same for IPv6: https://github.com/openconfig/featureprofiles/blob/e3dc053e7c0abe36b6899660c377b7f0bbe09632/feature/experimental/bgp/ate_tests/bgp_remove_private_as/bgp_remove_private_as_test.go#L131

Reply:
updated, thank you